src/server/scripts/Outland/HellfireCitadel/ShatteredHalls/instance_shattered_halls.cpp

```diff
@@ -20,6 +20,13 @@
 #include "ScriptMgr.h"
 #include "shattered_halls.h"
 
+ObjectData const creatureData[] =
+{
+    { NPC_GRAND_WARLOCK_NETHEKURSE  , DATA_NETHEKURSE     },
+    { NPC_WARCHIEF_KARGATH          , DATA_KARGATH        },
+    { 0,                                                  , 0                                }
```

Comment:
```suggestion
    { 0,                                                  0                                }
```

Reply:
You had me confused for a bit with the previous one. but yeah push it